x/ccv/provider/keeper/throttle.go

```diff
@@ -494,3 +494,54 @@ func (k Keeper) SetLastSlashMeterFullTime(ctx sdktypes.Context, time time.Time)
 	store := ctx.KVStore(k.storeKey)
 	store.Set(providertypes.LastSlashMeterFullTimeKey(), sdktypes.FormatTimeBytes(time.UTC()))
 }
+
+// GetThrottledPacketData fetches packet data from the store using consumerChainId and ibcSeqNum
+// Since multiple types can be stored, it is up to the caller to determine the type after fetching
+func (k Keeper) GetThrottledPacketData(ctx sdktypes.Context, consumerChainID string, ibcSeqNum uint64) interface{} {
```

Comment:
I think we can prob remove this in favor of `GetAllThrottledPacketData`, right? Would just have to change the comment about it only being used for tests

Reply:
`GetAllThrottledPacketData` returns 2 slices whereas this function returns a single array of all packets in exact order they will be applied.

With a single slice it is easy to see which maturedVSC is being held delayed due to preceding slash packets.

Reply:
`GetAllThrottledPacketData` returns a single array of all the packets in the order they'll be applied, see `rawOrderedData` return value. Does the original method still exist tho? Not seeing it now